packages/server/lib/clients/publisher.ts

```diff
@@ -0,0 +1,127 @@
+import type { WebSocket } from 'ws';
```

Comment:
can we keep the [pattern](https://github.com/NangoHQ/nango/tree/master/packages/server/lib/clients) that the word "client" is included in the name of the file?

`publisher.client.ts`